bootfiles/flatcar/bootfile.sh.tmpl

```diff
@@ -1,19 +1,46 @@
 #!/bin/bash
 
-cat <<EOF >/tmp/bootcloudinit.yaml
-#cloud-config
-variant: flatcar
-version: 1.0.0
-kernel_arguments:
-  should_not_exist:
-    - flatcar.autologin
-passwd:
-  users:
-    - name: core
-      ssh_authorized_keys:
-        - "${public_key}"
+cat <<EOF >/tmp/ignition.json
+{
+  "ignition": {
+    "config": {},
+    "security": {
+      "tls": {}
+    },
+    "timeouts": {},
+    "version": "2.3.0"
+  },
+  "networkd": {},
+  "passwd": {
+    "users": [
+      {
+        "name": "core",
+        "sshAuthorizedKeys": [
+          "${public_key}"
+        ]
+      }
+    ]
+  },
+  "storage": {},
+  "systemd": {
+    "units": [
+      {
+        "enable": true,
+        "name": "docker.service"
+      },
+      {
+        "mask": true,
+        "name": "update-engine.service"
+      },
+      {
+        "mask": true,
+        "name": "locksmithd.service"
+      }
+    ]
+  }
+}
 EOF
 
-flatcar-install -d /dev/sda -o vmware_raw -c /tmp/bootcloudinit.yaml
+flatcar-install -d /dev/sda -o vmware_raw -C lts -i /tmp/ignition.json
```

Comment:
```suggestion
flatcar-install -d /dev/sda -o vmware_raw -i /tmp/ignition.json
```

flatcar-install receives defaults like channel and version from the image itself. I would suggest not being overspecific as we may also want to work with other channels in the future and so we could reuse this script

WDYT?

Reply:
sounds great. If flatcar-install can figure out defaults and version, it would make it easier. I will remove this option.